### PR TITLE
Fixed `require('.\')`

### LINKED ISSUE #27: require('./') not working
`require('./')` is supposed to be resolved as `./index.js` but is resolved as `.//index.js` instead.

### PR DESCRIPTION
**Pull request by benweet**
Fixes #27

### CODE REVIEW

File: lib/require.js [hunk 1]
```diff
@@ -44,12 +44,13 @@ require.resolve = function (path) {
   // GH-12
   if ('.' != path[0]) return native_require.resolve(path);
 
+  pathWithSlash = path.slice(-1) === '/' ? path : path + '/';
```
Comment (stephenmathieson) [inline lib/require.js]: missing `var` ... no need for this to be global

**Comment by stephenmathieson** [comment]
add a test please?

**Comment by benweet** [comment]
Just adapted an existing one..

**Comment by stephenmathieson** [comment]
awesome, thanks!

**Comment by stephenmathieson** [comment]
0.5.4 :)

**Pull request merged**